Document new parameter `min_ipv6_addr_count` and sub sensors for wifi_info and ethernet_info (#3234)
OttoWinter/esphomedocs: components/network.rst

    # Example configuration
    network:
        enable_ipv6: true
        min_ipv6_addr_count: 2
        
Configuration variables:
------------------------

- **enable_ipv6** (*Optional*, boolean): Enables IPv6 support. Defaults to ``false``.

> - **min_ipv6_addr_count** (*Optional*, integer): ESPhome considers network connected when it has gotten one IPv4 address and this amount of IPv6 addresses. Defaults to ``0`` to not hang on boot with networks with no IPv6 enabled. ``2`` could be a good value for a configuration that requires IPv6.

Reviewer: ```suggestion
- **min_ipv6_addr_count** (*Optional*, integer): ESPhome considers the network to be connected when it has one IPv4 address and this number of IPv6 addresses. Defaults to ``0`` so as to not hang on boot with networks where IPv6 is not enabled. ``2`` is typically a reasonable value for configurations requiring IPv6.
```
